### PR TITLE
partial warning

### LINKED ISSUE #5938: Partial tag helper analyzer warnings
See [5:20 in video](https://www.dropbox.com/s/9olqhldt75vngoz/Review%20ASP.NET%20Core%202.1%20doc%20priorities%20-%20Monday%2C%20April%209%2C%202018%208.09.35%20AM.mp4?dl=0)  

Analyzer flags HTML.Partial sync version saying you should use the async version or the [Partial Tag Helper](https://docs.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/built-in/partial-tag-helper?view=aspnetcore-2.1&viewFallbackFrom=aspnetcore-2.0)

Customers will want to know what to do.

@guardrex  @scottaddie  where should this be documented? How do I run the analyzer?

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #5938 
image

### CODE REVIEW

File: aspnetcore/testing/troubleshoot.md [hunk 1]
```diff
@@ -60,4 +60,16 @@ In the **New Project** dialog for ASP.NET Core you may see the following warning
 This warning appears when the environment variable `PATH` doesn’t point to any .NET Core SDKs on the machine. To resolve this problem:
 
 * Install or verify the .NET Core SDK is installed.
-* Verify the `PATH` environment variable points to the location the SDK is installed. The installer normally sets the `PATH`.
\ No newline at end of file
+* Verify the `PATH` environment variable points to the location the SDK is installed. The installer normally sets the `PATH`.
+
+::: moniker range=">= aspnetcore-2.1"
+
+### Use of IHtmlHelper.Partial may result in application deadlocks.
```
Comment (guardrex) [inline aspnetcore/testing/troubleshoot.md]: No period on section header

File: aspnetcore/testing/troubleshoot.md [hunk 2]
```diff
@@ -60,4 +60,16 @@ In the **New Project** dialog for ASP.NET Core you may see the following warning
...
\ No newline at end of file
+* Verify the `PATH` environment variable points to the location the SDK is installed. The installer normally sets the `PATH`.
+
+::: moniker range=">= aspnetcore-2.1"
+
+### Use of IHtmlHelper.Partial may result in application deadlocks. 
+
+In ASP.NET Core 2.1 and later, calling `Html.Partial` will result in an analyzer warning due to the potential for deadlocks. The warning message is:
+
+*Use of IHtmlHelper.Partial may result in application deadlocks. Consider using `<partial>` Tag Helper or `IHtmlHelper.PartialAsync`.
```
Comment (guardrex) [inline aspnetcore/testing/troubleshoot.md]: Italics are started (`*`) but not ended. Missing a closing asterisk?

File: aspnetcore/testing/troubleshoot.md [hunk 3]
```diff
@@ -60,4 +60,16 @@ In the **New Project** dialog for ASP.NET Core you may see the following warning
...
 * Install or verify the .NET Core SDK is installed.
-* Verify the `PATH` environment variable points to the location the SDK is installed. The installer normally sets the `PATH`.
\ No newline at end of file
+* Verify the `PATH` environment variable points to the location the SDK is installed. The installer normally sets the `PATH`.
+
+::: moniker range=">= aspnetcore-2.1"
+
+### Use of IHtmlHelper.Partial may result in application deadlocks. 
+
+In ASP.NET Core 2.1 and later, calling `Html.Partial` will result in an analyzer warning due to the potential for deadlocks. The warning message is:
```
Comment (guardrex) [inline aspnetcore/testing/troubleshoot.md]: Remove future tense: "will result in" 👉 "results in"

**Comment by scottaddie** [comment]
@Rick-Anderson Should this be moved to the following doc instead? https://docs.microsoft.com/aspnet/core/mvc/views/partial

**Comment by Rick-Anderson** [comment]
Good idea. I'll do that.

**Pull request merged**